tools/flang2/flang2exe/expsmp.c

```diff
@@ -728,7 +728,7 @@ setTaskloopVars(int lb, int ub, int stride, int lastitr)
   }
 
   if (lastitr && STYPEG(lastitr) != ST_CONST) {
-    nme = addnme(NT_IND, st, basenm, 0);
```

Comment:
Looks like a bug fix?

Reply:
Sorry. I accidentally deleted the commit message related to this change. Here it is for clarity:

Fix typo, should pass lastitr which is symbol to addnme